src/main/java/stream/flarebot/flarebot/commands/music/SeekCommand.java

```diff
@@ -15,24 +17,17 @@
 import stream.flarebot.flarebot.util.GeneralUtils;
 import stream.flarebot.flarebot.util.MessageUtils;
 
+
 public class SeekCommand implements Command {
 
     @Override
     public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
         if (args.length == 1) {
-            PeriodFormatter formatter = new PeriodFormatterBuilder()
-                    .appendHours().appendSuffix("h")
-                    .appendMinutes().appendSuffix("m")
-                    .appendSeconds().appendSuffix("s")
-                    .toFormatter();
-            Period period;
-            try {
-                period = formatter.parsePeriod(args[0]);
-            } catch (IllegalArgumentException e) {
-                MessageUtils.sendErrorMessage("The duration is not in the correct format!", channel);
+            Long millis = GeneralUtils.parseTime(args[0]);
+            if (millis == null || millis > Integer.MAX_VALUE) {
```

Comment:
Why not just make this get handled like in getInt ?

getInt has 2 params, the string and a default return value. If something like this happens it uses the default value, just use that and 0

Reply:
:+1: